bun/src/binding.ts

```diff
@@ -116,10 +113,15 @@ const lib = dlopen(binaryPath, {
 		],
 		returns: "u32",
 	},
-});
+};
 
-export const binding = lib.symbols;
+// Load the compiled Zig shared library
+const lib = await openLibrary(path.resolve("../bun"), fns);
+export const binding: ConvertFns<typeof fns> = lib.symbols;
```

Comment:
I don't think you need the explicit type here. openLibrary should pass through the proper type to lib.symbols